.travis.yml

```diff
@@ -1,9 +1,22 @@
 language: python
 python:
-  - 2.7
```

Comment:
Yes, it's how @lampslave explained.. people prefer keeping the Python matrix in tox.ini, so it's *not* specific to the CI conf, as you should be able to run the specification both locally.. or switch CI for that matter.